### PR TITLE
Documentation for Datadog Multi-Query Support

### PR DESCRIPTION
**Pull request by dgibbard-cisco**
Signed-off-by: Darren Gibbard <dgibbard@cisco.com>

Documentation to support the Datadog Multi-Query Support added in https://github.com/kedacore/keda/pull/3550

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes [#3423](https://github.com/kedacore/keda/issues/3423)

### CODE REVIEW

**Comment by dgibbard-cisco** [comment]
Would be good to get @arapulido 's eyes on this too if available, as the main code creator :)

File: content/docs/2.9/scalers/datadog.md [hunk 1]
```diff
@@ -137,3 +140,65 @@ which by default is 15 seconds. For example, if the `kube-controller-manager`
 was started with `--horizontal-pod-autoscaler-sync-period=30`, the HPA will poll
 Datadog for a metric value every 30 seconds while the number of replicas is
 between 1 and N.
+
+## Multi-Query Support
+
+To reduce issues with API Rate Limiting from Datadog, it is possible to send a single query, which contains multiple queries, comma-seperated.
```
Comment (tomkerkhove) [inline content/docs/2.9/scalers/datadog.md:146]:
```suggestion
To reduce issues with API rate limiting from Datadog, it is possible to send a single query, which contains multiple queries, comma-seperated.
```

File: content/docs/2.9/scalers/datadog.md [hunk 2]
```diff
@@ -108,6 +109,8 @@ spec:
       query: "sum:trace.redis.command.hits{env:none,service:redis}.as_count()"
       # Required: according to the number of query result, to scale the TargetRef
       queryValue: "7"
+      # Optional: How to aggregate results for multi-query queries. Default: 'max'
+      queryAggregator: "max"
```
Comment (tomkerkhove) [inline content/docs/2.9/scalers/datadog.md:113]: Can you add this to the parameter list above please?

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
LGTM, a few nits

**Comment by dgibbard-cisco** [comment]
> LGTM, a few nits

Thanks for the review! Have made the changes as suggested 👍

**Review by tomkerkhove — APPROVED** [review]

**Review by arapulido — CHANGES_REQUESTED** [review]
Small suggestion. Other than that, LGTM

File: content/docs/2.9/scalers/datadog.md [hunk 3]
```diff
@@ -137,3 +141,65 @@ which by default is 15 seconds. For example, if the `kube-controller-manager`
 was started with `--horizontal-pod-autoscaler-sync-period=30`, the HPA will poll
 Datadog for a metric value every 30 seconds while the number of replicas is
 between 1 and N.
+
+## Multi-Query Support
+
+To reduce issues with API rate limiting from Datadog, it is possible to send a single query, which contains multiple queries, comma-seperated.
+When doing this, the results from each query are aggregated based on the `queryAggregator` value (eg: `max` or `average`).
```
[the remark this answers is not in the document]
Comment (dgibbard-cisco) [reply content/docs/2.9/scalers/datadog.md:146]: Have added that, and also updated the overall documentation on multi-query to be as clear as possible.

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**